riscv/plic.cc

```diff
@@ -340,17 +340,15 @@ bool plic_t::load(reg_t addr, size_t len, uint8_t* bytes)
     }
   }
 
-  if (ret) {
```

Comment:
The removal of the conditional here should be its own commit, so in case it happens to cause some problem, `git bisect` can pinpoint it.

Reply:
Ack.